src/index.html

```diff
@@ -45,67 +51,121 @@
+    <meta name="apple-mobile-web-app-title" content="SINFO" />
+    <meta name="application-name" content="SINFO" />
+    <meta name="theme-color" content="#ffffff" />
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300,300italic,400italic,500italic&subset=latin,latin-ext"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://fonts.googleapis.com/css?family=PT+Sans+Caption:400,700"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://unpkg.com/@angular/material/prebuilt-themes/indigo-pink.css"
+      rel="stylesheet"
+    />
+
+    <script src="https://accounts.google.com/gsi/client"></script>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=UA-59828816-1"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "UA-59828816-1");
     </script>
-  <script type="text/javascript" src="assets/js/ventcamp.js"></script>
-  <script type="text/javascript" src="https://connect.facebook.net/en_US/sdk.js"></script>
-  <script src="https://apis.google.com/js/api.js"></script>
 
-  <script id="mcjs">
-    ! function (c, h, i, m, p) {
-      m = c.createElement(h), p = c.getElementsByTagName(h)[0], m.async = 1, m.src = i, p.parentNode.insertBefore(m, p)
-    }(document, "script",
-      "https://chimpstatic.com/mcjs-connected/js/users/de57247db0922adf365af7eef/e50b1f878d1ef5acaaea91fcc.js");
-
-  </script>
-</body>
-
-</html>
\ No newline at end of file
+    <!--[if lt IE 9]>
+      <script src="assets/js/html5shiv.js"></script>
+      <script src="assets/js/respond.min.js"></script>
+    <![endif]-->
+  </head>
+
+  <body data-spy="scroll" data-offset="100">
+    <app-root></app-root>
+
+    <script
+      type="text/javascript"
+      src="https://app.sinfo.org/assets/js/jquery-2.1.4.min.js"
+    ></script>
+    <script
+      src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
+      integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+      crossorigin="anonymous"
+    ></script>
+    <script type="text/javascript" src="assets/js/ventcamp.js"></script>
+    <script
+      type="text/javascript"
+      src="https://connect.facebook.net/en_US/sdk.js"
```

Comment:
I think we can also get rid of this, since we do not use Facebook authentication anymore, right?